scripts/push-all.sh

```diff
@@ -55,7 +55,7 @@ current_build_tags() {
 # HACK : unconditionally push to latest
 #DOCKER_TAG="${DOCKER_TAG} latest"
 
-current_build_tags | grep -Ev 'akamai/(base|cli|.*-chain)' |
+current_build_tags | grep -Ev 'akamai/(base|.*-chain)' |
```

Comment:
@javiergarza could you please confirm that the dockerhub credentials we are using will work with the akamai/cli repository?

Reply:
I believe so, I can try to keep a look at it tomorrow to ensure the build succeeds